templates/project-ts/src/index.test.ts

```diff
@@ -1,3 +1,5 @@
+import _Add from './index';
```

Comment:
Does it work if this imports `Add`? (I think the dummy one might confuse people.) 

Long term solution is to add the actual test, of course.

Reply:
It does, but TS complains that it's an unused import. There is no warning in the build process so I think it'll probably be better if it's just `Add` like you suggest.

Reply:
okay, yeah that sounds good. Then someday we'll add the actual test